manual/src/cmds/runtime-tracing.etex

```diff
@@ -15,6 +15,10 @@ Data emitted includes:
 \item Allocation and promotion rates between heaps
 \end{itemize}
 
+ Additional events can be declared and
+consumed, providing higher-level dynamic introspection capabilities to OCaml
```

Comment:
Nit: I would say "monitoring" (or maybe "observability") rather than "introspection", and I am not sure what "dynamic" means and brings in this context. (Also there is a weird space at the beginning of line 18?)

Reply:
Addressed in 4538ac4728.